libexec/rc/rc.d/ldconfig

```diff
@@ -43,6 +43,25 @@ ldconfig_start()
 		machine_arch=$(sysctl -n hw.machine_arch)
 
 		case ${machine_arch} in
+		aarch64c|riscv64c)
+			for i in ${ldconfig_local64_dirs}; do
+				if [ -d "${i}" ]; then
+					_files=`find ${i} -type f`
+					if [ -n "${_files}" ]; then
+						ldconfig64_paths="${ldconfig64_paths} `cat ${_files} | sort -u`"
+					fi
+				fi
+			done
+			_LDC=""
+			for i in ${ldconfig64_paths}; do
+				if [ -r "${i}" ]; then
+					_LDC="${_LDC} ${i}"
+				fi
+			done
+			check_startmsgs &&
+			    echo '64-bit compatibility ldconfig path:' ${_LDC}
+			${ldconfig} -elf -m ${_ins} ${_LDC}
```

Comment:
This looks dodgy, isn't this going to touch the native ABI's hints? Having -elf and -m here when not needed for ldconfig32 seems dubious too.

Reply:
Yes, you're right. I just tested it and it requires a separate hints file. I added `-64` to `ldconfig` and will update this PR once I make sure it works.

Reply:
It should work now.